specification/indices/stats/types.ts

```diff
@@ -176,4 +178,10 @@ export class ShardStats {
   translog: TranslogStats
   warmer: WarmerStats
   bulk?: BulkStats
+  /** @since 7.15.0 */
+  shards?: ShardsTotalCount
+}
+
+export class ShardsTotalCount {
```

Comment:
In the interest of future additions should we name this differently? Maybe `ShardsTotalStats`/`ShardsAllStats`?

Also is this field optional in 7.15+ or is it always there?

Reply:
@sethmlarson I agree, can you finish up this pr? Thanks!